<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Geometry</class>
 <widget class="QDialog" name="Geometry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Mesh</string>
  </property>
  <widget class="QTableWidget" name="inforTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>411</width>
     <height>481</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Layer #</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Thickness (m)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Material #</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sub-layer</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/imgs/geometry.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>411</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QComboBox" name="analysisBox">
      <item>
       <property name="text">
        <string>1- No Smear - No Well Resistance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2- No Smear - Well Resistance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3- Smear - No Well Resistance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4- Smear - Well Resistance </string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="meshButton">
        <property name="text">
         <string>Create Mesh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setSubLayer">
        <property name="text">
         <string>Set Sub-layer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="replotButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>130</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Unit cell Radius (m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Smear-zone Radius (m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>PVD Radius (m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>PVD Length (m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Suraface elevation (m)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="reLine">
          <property name="text">
           <string>0.03175</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rsLine">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rwLine">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lengthLine">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="elevationLine">
          <property name="text">
           <string>0.0254</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Number of Layer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Smear-zone elements</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Undisturbed-zone elements</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Discharge capacity (m3/s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Default Sub-Layer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="layerLine">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="npsLine">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nplLine">
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qwLine"/>
        </item>
        <item>
         <widget class="QLineEdit" name="subLayerLine">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
